config/core/test-configs.yaml

```diff
@@ -2130,6 +2137,19 @@ test_configs:
       - baseline
       - baseline-nfs
 
+  - device_type: juno-r0-acpi
+    test_plans:
+      - baseline
+      - baseline-nfs
+      - kselftest-futex
+      - kselftest-lib
+      - kselftest-lkdtm
```

Comment:
Some failures: https://staging.kernelci.org/test/plan/id/623d235ced17b24213520901/

Reply:
OK, and this is not a KernelCI problem per se but some thing to fix either in the test-definitions skiplist or in kselftest upstream.